CHANGELOG.asciidoc

```diff
@@ -40,6 +40,10 @@ Notes:
 [float]
 ===== Bug fixes
 
+* <<span-to-string,`span.toString()`>> and <<transaction-to-string,`transaction.toString()`>>
+  have been *deprecated*. The exact string output may change in v4 of the
+  agent.
+
 * Improve `span.sync` determination (fixes {issue}1996[#1996]) and stop
   reporting `transaction.sync` which was never used ({issue}2292[#2292]).
```

Comment:
Not a part of this PR, so idk if you want to fix it now, but this attribute is incorrect.

```suggestion
* Improve `span.sync` determination (fixes {issues}1996[#1996]) and stop
  reporting `transaction.sync` which was never used ({issues}2292[#2292]).
```

Reply:
Thanks! Also in commit 328be4c0